readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: ecommerce, e-commerce, store, sales, reports, analytics, dashboard, activi
 Requires at least: 5.3.0
 Tested up to: 5.3.2
 Requires PHP: 5.6.20
-Stable tag: 1.2.1
+Stable tag: 1.2.2
```

Comment:
@psealock Can we bump the `Tested up to` version as well?

Reply:
Odd, that was done in https://github.com/woocommerce/woocommerce-admin/commit/3a08e387b62f7f24df8b6dd9f8f67c6402a99b5c .

Reply:
Done in https://github.com/woocommerce/woocommerce-admin/pull/4395/commits/714cbf984dfc226901ff23026453796fa001ca84